{
    "type": "bundle",
    "id": "bundle--8d37c404-b181-4f5c-a4da-65f8f1a88d49",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--26818dac-707e-4fcd-9e8a-1a413633146c",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-06-24T00:05:49.810426Z",
            "modified": "2023-06-24T00:05:49.810426Z",
            "name": "CVE-2023-32371",
            "description": "The issue was addressed with improved checks. This issue is fixed in iOS 16.5 and iPadOS 16.5, macOS Ventura 13.4. An app may be able to break out of its sandbox",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-32371"
                }
            ]
        }
    ]
}